HyperVExtension/src/HyperVExtension/Helpers/BytesHelper.cs

```diff
@@ -1,12 +1,18 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System;
 using System.Globalization;
+using Windows.Win32;
+using Windows.Win32.Foundation;
+using Windows.Win32.UI.Shell;
 
 namespace HyperVExtension.Helpers;
 
 public static class BytesHelper
 {
+    private const int ByteToStringBufferSize = 16;

```

Comment:
Seems like a platform dependent number.  Are you sure this will always be 16?

Reply:
actually its not platform dependent, interestingly 16 is the amount of characters the buffer that the Windows Settings storage page uses when using the StrFormatByteSizeEx api. So we're just doing the same thing. Works pretty good